utility/utils.py

```diff
@@ -1148,6 +1150,13 @@ def __init__(self):
                 )
             except BaseException:  # noqa
                 log.warning("Unable to connect to Report Portal.")
+        if tfacon_cfg:
+            self.is_tfacon_cfg = True
+            self.project_name = tfacon_cfg.get("project_name")
+            self.auth_token = tfacon_cfg.get("auth_token")
+            self.platform_url = tfacon_cfg.get("platform_url")
+            self.tfa_url = tfacon_cfg.get("tfa_url")
+            self.connector_type = tfacon_cfg.get("connector_type")
```

Comment:
IMO, this can be avoided instead do the entire check during `finish_launch`. The configuration is required in that method alone.